src/components/operations/operation-details/ko/runtime/operation-console.html

```diff
@@ -247,9 +247,24 @@ <h3>Headers</h3>
                     <!-- /ko -->
                     <!-- ko if: !header.options || header.options.length === 0 -->
                     <div class="input-group">
+                        <!-- ko if: !header.secret -->
                         <input type="text" autocomplete="off" class="form-control form-control-sm" placeholder="value"
                             spellcheck="false" aria-label="Header value"
                             data-bind="event: { keyup: $component.updateRequestSummary }, textInput: header.value">
+                        <!-- /ko -->
+                        <!-- ko if: header.secret -->
+                        <div class="input-div">
```

Comment:
Regarding `input-div` and `password-input` classes: it looks very similar to what `.input-group` was intended to do. We already have this class defined in forms.scss of website styles, but it is outdated a bit. Please take a look at [naming and structure](https://getbootstrap.com/docs/5.1/forms/input-group/) of Bootstrap for input groups, I would even steal those names from them. This would allow us to have common utility to complex inputs like your password-input, and at the same time we don't introduce a new CSS class for each such case.

You may have a question like "why don't we just use the whole Bootstrap library?" :) So, answering it in advance: This is because we have to generate component styles and their variations on the fly. However, in many ways we use similar approach to structuring and naming our generated classes, in which the Bootstrap team makes a very good job.